gcovr/tests/shadow/Makefile

```diff
@@ -0,0 +1,29 @@
+all:
+	cd _build && $(MAKE) -f Makefile all
```

Comment:
Use directory build and add a `.gitignore` with the content:
```
!build
``` 

or use `output` instead of `build`.

Reply:
done